packages/router/src/grpc/view-protocol-server/app-parameters.test.ts

```diff
@@ -30,6 +30,7 @@ describe('AppParameters request handler', () => {
       method: ViewService.methods.appParameters,
       protocolName: 'mock',
       requestMethod: 'MOCK',
+      url: '/mock',
```

Comment:
url is a required field, IDE highlighted it as an error, although all checks passed